examples/standalone-data-generator/application/shopping/ScreenEvent.py

```diff
@@ -88,6 +89,7 @@ def get_exit_app_events(user, event):
             exception_event["attributes"]["_exception_message"] = exception[0]
             exception_event["attributes"]["_exception_stack"] = exception[1]
             events.append(ShoppingApp.get_final_event(user, EventType.APP_EXCEPTION, exception_event))
+            print("record exception event")
```

Comment:
It will generate lots of outputs on console.

If you need verbose logging, let's use a logger with a debug level for those kinds of logs.

Reply:
deleted.